cli/cmd/chunkIntervalReset.go

```diff
@@ -26,7 +28,20 @@ func chunkIntervalReset(cmd *cobra.Command, args []string) error {
 
 	metric := args[0]
 
-	pool, err := pgconn.OpenConnectionToDB(namespace, name, user, dbname, FORWARD_PORT_TSDB)
+	secretKey, user, err := utils.GetDBSecretKeyAndDBUser(helmReleaseName, user)
+	if err != nil {
+		return fmt.Errorf("could not get DB secret key from gelm release: %w", err)
+	}
+
+	d := pgconn.DBDetails{
+		Namespace: namespace,
+		Name:      helmReleaseName,
+		DBName:    dbname,
+		User:      user,
+		SecretKey: secretKey,
+		Remote:    FORWARD_PORT_TSDB,
+	}
```

Comment:
I think there's a possibility to abstract these lines in a function, I think they are the same throughout the code (e.g. here and chunkIntervalGet.go lines 32-44)